package.json

```diff
@@ -38,25 +39,50 @@
     "karma-jasmine": "1.1.0",
     "lodash": "4.17.4",
     "mousetrap": "1.4.6",
-    "neutrino": "4.3.1",
-    "neutrino-lint-base": "4.3.1",
-    "neutrino-preset-karma": "4.2.0",
-    "neutrino-preset-react": "4.2.3",
+    "neutrino": "^5.2.0",
+    "neutrino-middleware-copy": "^5.2.0",
+    "neutrino-middleware-eslint": "^5.2.0",
+    "neutrino-preset-karma": "^5.2.0",
+    "neutrino-preset-react": "^5.2.0",
     "ngreact": "0.3.0",
     "raw-loader": "0.5.1",
     "react": "15.3.1",
     "react-addons-test-utils": "15.3.1",
     "react-dom": "15.3.1",
     "taskcluster-client": "1.6.3",
+    "webpack": "^2.3.2",
```

Comment:
webpack will be pulled in via the neutrino dependencies. What are your thoughts about leaving neutrino to set the version instead? The versions will still be pinned via `yarn.lock`.